venter/tests.py

```diff
@@ -1,7 +1,14 @@
+"""
+To run the tests fire this command:
+
+coverage run manage.py test venter --settings backend.settings_test && coverage html
```

Comment:
No meta information in source code please. If this command changes later, this change will stay and confuse the next person. Instead, make a separate PR to add this to README.